frontend/src/lib/components/flows/content/FlowModule.svelte

```diff
@@ -44,6 +44,23 @@
 	let websocketAlive = { pyright: false, black: false, deno: false }
 	let selected = 'inputs'
 
+	$: if (editor) {
+		editor.addAction('insert-variable', 'Windmill: Insert variable', (ed) => {
+			const content = document.getElementById('insert-variable')
+			if (content) {
+				const buttons = content.getElementsByTagName('button')
+				buttons[0].click()
+			}
+		})
+		editor.addAction('insert-resource', 'Windmill: Insert resource', (ed) => {
+			const content = document.getElementById('insert-resource')
+			if (content) {
+				const buttons = content.getElementsByTagName('button')
+				buttons[0].click()
+			}
+		})
+	}
```

Comment:
1. It should also be in ScriptEditor so either we do it in EditorBar or in Editor directly (non relevant instances of the Editor use SimpleEditor anyway)
2. When you use $, it's easier to read to do `$: editor && myAction()` because otherwise one has to make sure the reactive statement doesn't also trigger on any change in a variable in the closure of the block

Reply:
Both points are done.